### PR TITLE
Add status delete check

### PR DESCRIPTION
**Pull request by nicolasThal**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1316
https://github.com/open-orchestra/open-orchestra-model-interface/pull/141
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/494

### CODE REVIEW

File: BackofficeBundle/Resources/config/usage_finder.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+parameters:
+    open_orchestra_backoffice.usage_finder.status.class: OpenOrchestra\Backoffice\UsageFinder\StatusUsageFinder
+
+services:
+    open_orchestra_backoffice.usage_finder.status:
+        class: %open_orchestra_backoffice.usage_finder.status.class%
+        calls:
+            - ['addRepository', [@open_orchestra_model.repository.node]]
+            - ['addRepository', [@open_orchestra_model.repository.content]]
+            - ['addRepository', [@open_orchestra_model.repository.role]]
```
Comment (Thiblef) [inline BackofficeBundle/Resources/config/usage_finder.yml]: eof

**Comment by Thiblef** [comment]
when tests run :+1:

**Comment by itkg-alavieille** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**